### PR TITLE
Update Hungarian translation to 1.15.6

### PR DESCRIPTION
**Pull request by smileyhead**
Hopefully I'm doing this correctly this time around.

### CODE REVIEW

File: resources/hu/cemu.po [hunk 1]
```diff
@@ -1,1333 +1,1780 @@
...
+"Language: hu\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.2.2\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: BreakpointWindow.cpp:41
+msgid "On"
+msgstr "Be"
```
Comment (Fs00) [inline resources/hu/cemu.po]: This should be translated as "Active", not as the "on" preposition.

File: resources/hu/cemu.po [hunk 2]
```diff
@@ -1,1333 +1,1780 @@
...
+"Az előre összeállított shaderek felgyorsíthatják a betöltés idejét a "
+"shaderbetöltési képernyőn.\n"
+"Az automatikus üzemmód engedélyezi AMD/Intel GPU-k esetén, viszont letiltja "
+"az NVIDIA GPU-knál egy illesztőprogramhiba kiküszöbölése érdekében.\n"
+"\n"
+"Javasolt érték: Automatikus"
+
+#: GameProfileWindow.cpp:105
+msgid "Shader mul accuracy"
+msgstr "Shader mul pontosság"
```
Comment (Fs00) [inline resources/hu/cemu.po]: In the English string "mul" is a shorthand for "multiplication". Is that immediately clear for an Hungarian reader? If not, use the Hungarian term for "multiplication".

File: resources/hu/cemu.po [hunk 3]
```diff
@@ -1,1333 +1,1780 @@
...
+#: GeneralSettings2.cpp:94
+msgid "Discord Presence"
+msgstr "Discord Presence"
+
+#: GeneralSettings2.cpp:96
+msgid ""
+"Enables the Discord Rich Presence feature\n"
+"You will also need to enable it in the Discord settings itself!"
+msgstr ""
+"Engedélyezi a Discord Rick Presence-funkcióját\n"
```
Comment (Fs00) [inline resources/hu/cemu.po]: Typo? ("_Rich_ Presence")

File: resources/hu/cemu.po [hunk 4]
```diff
@@ -1,1333 +1,1780 @@
...
+msgid "MLC Path"
+msgstr "MLC elérési út"
+
+#: GeneralSettings2.cpp:123
+msgid ""
+"Select a custom mlc path\n"
+"The mlc path is used to store Wii U related files like save games, game "
+"updates and dlc data"
+msgstr ""
+"Adj meg egy egyéni MLC  elérési utat\n"
```
Comment (Fs00) [inline resources/hu/cemu.po]: Typo here (double space)

File: resources/hu/cemu.po [hunk 5]
```diff
@@ -1,1333 +1,1780 @@
...
+msgid "&Tools"
+msgstr "&Eszközök"
+
+#: MainWindow.cpp:2226
+msgid "&Single-core interpreter"
+msgstr "&Egymagos interpreter"
+
+#: MainWindow.cpp:2227
+msgid "&Single-core recompiler (fast)"
+msgstr "&Egymagos recompiler  (gyors)"
```
Comment (Fs00) [inline resources/hu/cemu.po]: Typo (double space)

**Comment by smileyhead** [comment]
Fixed everything! Sorry for the late reply.

**Comment by Fs00** [comment]
Merged, thanks! Next time remember to create a new up-to-date fork before starting to edit.

**Pull request merged**